src/Components/ADT3DSceneBuilder/Internal/Shared/PanelForms.styles.ts

```diff
@@ -46,8 +46,8 @@ export const getPanelFormStyles = memoizeFunction(
                     padding: '0 16px 4px'
                 } as IStyle
             ],
-            pivot: [
-                classNames.pivot,
+            expandedSection: [
```

Comment:
nit: what would you think of `expandingSection` since it expands to fill the space.